skglm/tests/test_group.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import numpy as np
+from numpy.linalg import norm
+
+from skglm.penalties.block_separable import WeightedGroupL2
+from skglm.datafits.group import QuadraticGroup
+from skglm.solvers.group_bcd_solver import bcd_solver
+
+from skglm.utils import grp_converter, make_correlated_data
+from celer import GroupLasso, Lasso
+
+
+def _generate_random_grp(n_groups, n_features, shuffle=True):
```

Comment:
this is not seeded and makes the test non deterministic (for the shuffle, and for the choice)